src/main/resources/morph-hbz01-to-lobid.xml

```diff
@@ -1022,6 +1022,15 @@
 		</combine>
 		<call-macro name="addTypeCollection" field="@idzdb"/>
 		<call-macro name="setBackRdfSubject" field="025z[-1].a"/>
+		<combine name="~rdf:subject" value="${a}">
+			<data source="@idzdb" name="a">
+				<regexp match="(.*)" format="$[ns-zdb-services]${1}"/>
+			</data>
```

Comment:
No need for the regex because this is already done in line 1012.

Reply:
In line 1012 onwards the ZDB-URL is created for owl/sameAs. But it's not created to use elsewhere, so `@idzdb` still contains only the ZDB-Id and not the ZDB-URL. Therefore the regular expression is needed at line 1025ff to reference the newly built owl/sameAs-ZDB-URL.